### ISSUE TITLE
Serchlist - Class spelling error

### ISSUE THREAD

**Issue by samjam48** [facn6/week4-AyaKira#15]
You should not have spelling errors in your code. You can abbreviate things but this feels like sloppy writing and would look bad if someone viewed your code.

![image](https://user-images.githubusercontent.com/16625180/61855415-b3846000-aec8-11e9-85c8-9a0fb042177a.png)

**Comment by samjam48** [comment]
this sounds pedantic but it relates to the man's point yesterday about spelling errors in your CV. It is a bit confusing that you can abbreviate words and make up how you do that. but generally if you are missing one vowel in a group of vowels it is wrong.

e.g. 
search or srch.   not serch
country or cntry.   not cuntry
abbreviate or abbrvt.  not abrevite

**Comment by kira-abl** [comment]
@samjam48 Yes, I agree, but hey, we did not write abrevite!  Anyways, it is now corrected, thanks.
